web/ui/react-app/src/pages/targets/SeriesLabels.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Tooltip } from 'reactstrap';
+import SeriesLabel from './SeriesLabel';
+
+interface Labels {
+  [key: string]: string;
+}
+
+export interface SeriesLabelsProps {
+  discoveredLabels: Labels;
+  labels: Labels;
+}
+
+const formatLabels = (labels: Labels): string[] => Object.keys(labels).map(key => `${key}="${labels[key]}"`);
```

Comment:
You can omit the return type ```string[]``` since typescript infer it from the returning value

Reply:
I thought there was a eslint rule about specifying return types?

Reply:
<img width="26" alt="Screen Shot 2019-11-05 at 2 55 45 PM" src="https://user-images.githubusercontent.com/6402556/68249595-5fffd300-ffdc-11e9-8408-d9b8634b29aa.png">

Reply:
<img width="869" alt="Screen Shot 2019-11-05 at 2 55 51 PM" src="https://user-images.githubusercontent.com/6402556/68249647-7148df80-ffdc-11e9-9083-0efb38a39fe6.png">

Reply:
Yeah, turns out there is. I see your point though. On the other hand, it's nice to specify types

Reply:
FWIW we have a gazillion missing ones currently and are just ignoring those warnings. I think that's fine for now, but also not opposed to adding return types to get rid of the warnings.